### ISSUE TITLE
chunking is still broken

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ndio#111]
Still broadcasting into array error.  See below:

``` python


import numpy
import ndio.remote.neurodata as nd
from datetime import datetime

oo = nd()

token = 'bock11'
channel = 'image'

startTime = datetime.now()

# Get image properties

res = 6
image_size = oo.get_token_info(token)['dataset']['imagesize'][str(res)]
image_offset = oo.get_token_info(token)['dataset']['offset'][str(res)]
image_res = oo.get_token_info(token)['dataset']['voxelres'][str(res)]
print image_size
print image_offset
total_voxels = (image_size[0]-image_offset[0])*(image_size[1]-image_offset[1])*(image_size[2]-image_offset[2])

#image_res in z dim should be 1
pixel_dim = image_res[0]*0.004*image_res[1]*0.004*.045
total_volume = total_voxels*pixel_dim

print 'total volume: ' + str(round(total_volume)) + ' um^3'

imout = oo.get_volume(token, channel, 0, image_size[0], 0, image_size[1], 2917, 4156, resolution=res, memory=True)


```

**Comment by j6k4m8** [comment]
if only we had a way to indicate what version of ndio you were using

**Comment by wrgr** [comment]
ndio reports v0.1.9, pip reports v0.1.11.  See if you can reproduce?

Oh - is that what the template is for?

**Comment by j6k4m8** [comment]
fixed in #116!!
